apps/vr-tests-web-components/src/stories/label/label.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28861&commitId=5100d85ccb482942638f5599c2e363f85a223136&env=prod&clientType=FLUENTUI)


  <details><summary><b>Label</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Default - Dark Mode.normal.chromium.png|0| Added|
|Label.Default.normal.chromium.png|0| Added|
|Label.Disabled.normal.chromium.png|0| Added|
|Label.Required.normal.chromium.png|0| Added|
|Label.Size Large.normal.chromium.png|0| Added|
|Label.Size Medium.normal.chromium.png|0| Added|
|Label.Size Small.normal.chromium.png|0| Added|
|Label.Weight Regular.normal.chromium.png|0| Added|
|Label.Weight Semibold.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>